Calculator.java

```diff
@@ -0,0 +1,72 @@
+package task_DEV-2;
+
+import java.io.*;
+import java.util.*;
+
+public class Calculator {
+  public static void main(String[] args) throws Exception {
+    BufferedReader d = new BufferedReader(new InputStreamReader(System.in));
+    String inputString;
+      try {
+        System.out.println("Put your arithmetic expression. Allows math symbals, operations +,-,*,/ and brackets ():");
+        inputString = d.readLine();
+        inputString = method(inputString);
+        System.out.println(calculate(inputString));
+      } catch (Exception e) {
+        System.out.println(e.getMessage());
+      }
+  }
+
+
+  /**
+   * Convert string into reverse polish notation
+   * @param inputString Input row
+   * @return Output row in reverse polish notation
+   */
+  public static String method(String inputString) throws Exception {
```

Comment:
javadoc, do not use static, name is senseless, must be decomposed to smaller methods/classes